src/app/metadata/features/metadata-browser/metadata-browser.component.ts

```diff
@@ -65,20 +71,27 @@ export class MetadataBrowserComponent implements OnInit {
    * On init, define the default values of the search variables
    */
   ngOnInit(): void {
-    this.#updateMetadataServiceSearchTerms();
-  }
-
-  /**
-   * Syncs the data between this component and the search service and initiates a new search.
-   */
-  #performSearch(): void {
-    this.#updateMetadataServiceSearchTerms();
+    this.#loadSearchTermsFromRoute();
   }
 
   /**
-   * Pushes the local filters, search term and page setup to the search service.
+   * This function takes the current URL, determines the query parameters and sets them accordingly in the metadata search service.
    */
-  #updateMetadataServiceSearchTerms(): void {
+  #loadSearchTermsFromRoute(): void {
+    const { s, q, f, p } = this.#router.routerState.snapshot.root.queryParams;
+    this.pageSize = parseInt(p) || DEFAULT_PAGE_SIZE;
+    if (q) {
+      this.searchTerm = q;
+    }
```

Comment:
```suggestion
    this.searchTerm = q || '';
```

this allows resetting the search term if it was set previously